spec/unit/lib/cloud_controller/diego/docker/staging_completion_handler_spec.rb

```diff
@@ -43,6 +43,24 @@ module Docker
             expect(runner).to have_received(:start)
           end
 
+          context 'when it receives docker image' do
+            let(:docker_image_name) { '10.244.2.6:8080/generated_id:latest' }
+            let(:payload) do
+              {
+                  execution_metadata: '"{\"cmd\":[\"start\"]}"',
+                  detected_start_command: { web: 'start' },
+                  lifecycle_data: { docker_image: docker_image_name }
```

Comment:
Maybe add a test for the case when `lifecycle_data: {}`? Also, what should the behavior be when `docker_image` is `""`?